src/drivers/ins/microstrain/module.yaml

```diff
@@ -0,0 +1,257 @@
+module_name: MICROSTRAIN
+parameters:
+    - group: Sensors
+      definitions:
+
+        MS_MODE:
+            description:
+                short: Toggles using the device as the primary EKF
+                long: |
+                    Setting to 1 will publish data from the device to the vehicle topics (global_position, attitude, local_position, odometry), estimator_status and sensor_selection
+                    Setting to 0 will publish data from the device to the external_ins topics (global position, attitude, local position)
+                    Restart Required
+            type: int32
+            default: 1
+
+        MS_IMU_RATE_HZ:
+            description:
+                short: IMU Data Rate
+                long: |
+                    IMU (Accelerometer and Gyroscope) data rate
+                    The INS driver will be scheduled at a rate 2*MS_IMU_RATE_HZ
+                    Max Limit: 1000
+                    0 - Disable IMU datastream
+                    The max limit should be divisible by the rate
```

Comment:
This is a little confusing. I would include an example of what you mean here. This applies to all data rates